apps/docsite/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@alma-oss/spirit-docsite",
+  "version": "0.0.0",
+  "private": true,
+  "type": "module",
+  "scripts": {
+    "dev": "next dev --turbopack",
+    "build": "next build",
+    "start": "next start"
+  },
+  "dependencies": {
+    "@lmc-eu/spirit-design-tokens": "workspace:^",
+    "@lmc-eu/spirit-icons": "workspace:^",
+    "@lmc-eu/spirit-web": "workspace:^",
+    "@lmc-eu/spirit-web-react": "workspace:^",
+    "next": "15.5.6",
+    "react": "19.0.0",
+    "react-dom": "19.0.0",
+    "sass": "1.85.1",
+    "sass-embedded": "1.85.1"
+  },
```

Comment:
- Some package versions can be updated: React 19.2, Next 16, Sass 1.93
- Do we need both `sass` and `sass-embedded`?
- Anyway, I am aware you are committing your work that is far from here, but don't we add the scripts and dependencies gradually as we need them?